src/test/isolation2/expected/resgroup/resgroup_startup_memory.out

```diff
@@ -0,0 +1,60 @@
+-- start_ignore
+! gpconfig -c runaway_detector_activation_percent -v 10;
+! gpstop -rai;
+
+drop table if exists t1;
+drop role if exists test;
+drop resource group test_group;
+-- end_ignore
+
+create resource group test_group with (cpu_rate_limit=20, memory_limit=15, memory_shared_quota=100, memory_spill_ratio=0);
+CREATE
+
+create role test with resource group test_group;
+CREATE
+
+set role test;
+SET
+create table t1 (a int) distributed by (a);
+CREATE
+insert into t1 select a from generate_series(1, 10) a;
+INSERT 10
+alter table t1 set distributed randomly;
+ALTER
+
+-- Test that the starting memory is visible to the resource group.
+1: set role test;
+SET
+1&: select count(*) from t1 where pg_sleep(1) is not null;  <waiting ...>
+
+2: select segment, mem.* from gp_toolkit.gp_resgroup_status, json_object_keys(memory_usage) as segment, json_to_record(memory_usage -> segment) mem (used int) where rsgname = 'test_group';
+ segment | used 
+---------+------
+ -1      | 12   
+ 0       | 12   
+ 1       | 12   
+ 2       | 12   
+(4 rows)
+1<:  <... completed>
+ count 
+-------
+ 10    
+(1 row)
+1q: ... <quitting>
+2q: ... <quitting>
+
+-- The runaway detector test. A query with a large number of slices should
+-- be terminated due to high memory consumption.
+select count(*) from t1 a1 join t1 a2 using(a) join t1 a3 using(a) join t1 a4 using(a) join t1 a5 using(a) join t1 a6 using(a) join t1 a7 using(a) join t1 a8 using(a) join t1 a9 using(a) join t1 a10 using(a);
+ERROR:  Canceling query because of high VMEM usage. current group id is 712716, group memory usage 133 MB, group shared memory quota is 102 MB, slot memory quota is 0 MB, global freechunks memory is 277 MB, global safe memory threshold is 277 MB (runaway_cleaner.c:197)  (seg1 slice10 172.18.0.3:6003 pid=88018) (runaway_cleaner.c:197)
```

Comment:
Rewrite please the test so that the error about memory consumption before the patch and after the patch occurs for different reasons